packages/@jsii/go-runtime/jsii-experimental/runtime.go

```diff
@@ -38,7 +39,7 @@ func Load(name string, version string, tarball []byte) {
 
 // Create will construct a new JSII object within the kernel runtime. This is
 // called by jsii object constructors.
-func Create(fqn FQN, args []interface{}, interfaces []FQN, overrides []Override, returns interface{}) {
+func Create(fqn FQN, args []interface{}, interfaces []FQN, overrides []Override, returnsPtr interface{}) {
```

Comment:
returnsPtr is `interface{}` because it can be a pointer to any type. Its essentially "the pointer to the value the calling function returns" which is an unknown type at this point.